samples/multi-agent/src/main/java/demo/multiagent/api/ActivityEndpoint.java

```diff
@@ -0,0 +1,93 @@
+package demo.multiagent.api;
+
+import akka.http.javadsl.model.HttpResponse;
+import akka.javasdk.annotations.Acl;
+import akka.javasdk.annotations.http.Get;
+import akka.javasdk.annotations.http.HttpEndpoint;
+import akka.javasdk.annotations.http.Post;
+import akka.javasdk.client.ComponentClient;
+import akka.javasdk.http.HttpResponses;
+import demo.multiagent.application.ActivityView;
+import demo.multiagent.application.AgentTeam;
+import demo.multiagent.application.PreferencesEntity;
+
+import java.util.List;
+import java.util.UUID;
+
+// Opened up for access from the public internet to make the service easy to try out.
+// For actual services meant for production this must be carefully considered, and often set more limited
+@Acl(allow = @Acl.Matcher(principal = Acl.Principal.INTERNET))
+@HttpEndpoint()
+public class ActivityEndpoint {
```

Comment:
I'm not sure if that's a good idea to rename it to Activity. 

The whole idea is to be multi functionality. For instance, I can simply ask for the weather without asking for an activity. And I can ask for an activity without mentioned weather in which case the `WeatherAgent` is not even selected.

Agree that `MultiAgentEndpoint` isn't great, but we can call it `ChatEndpoint` instead.

Reply:
I understand, but it becomes too generic for the first parts of the guide. There it is concretely about activities. Then in the last part it speaks about the dynamic planning and how that can be expanded to "anything".